nmsg/kafkaio.c

```diff
@@ -215,6 +230,19 @@ _kafka_set_state(kafka_ctx_t ctx, const char *func, kafka_state state)
 	ctx->state = state;
 }
 
+static void
+_kafka_set_connection_state(kafka_ctx_t ctx, const char *func, bool state)
```

Comment:
For legibility, I would recommend making this:
```
static void
_kafka_set_connected(kafka_ctx_t ctx, const char *func, bool connected)
```

Reply:
Fixed

Reply:
I am not seeing any change

Reply:
I think you are commenting on the wrong thread. I already marked this resolved